plugins/test/unit/plugins/distributors/test_configuration.py

```diff
@@ -140,7 +139,7 @@ def test__is_valid_repo_registry_id(self):
             '134567890',
             'alpha-numeric_123',
             'periods.are.cool',
-            '..............',
+            'repo-registry-id'
```

Comment:
Oh, we'll also want to include a registry ID with length 255 that should be valid. I think that's it, though.